pkg/quack/quack.go

```diff
@@ -208,6 +208,16 @@ func getTemplateInput(data []byte) ([]byte, error) {
 		}
 	}
 
+	for _, path := range ignoredPaths {
+		patch := []byte(fmt.Sprintf(`[
+			{"op": "remove", "path": "%s"}
+		]`, path))
+		data, err = applyPatch(data, patch)
+		if err != nil {
```

Comment:
```
$ kubectl --context=iapetus create -f alerts.yaml 
Error from server (InternalError): error when creating "alerts.yaml": Internal error occurred: admission webhook "quack.pusher.com" denied the request: Error creating template input: error removing /metadata/annotations/faros.pusher.com~1last-applied-configuration: unable to apply patch: Unable to remove nonexistent key: faros.pusher.com/last-applied-configuration
```

@BenPope ran into an issue, I think we need an exception here, @mthssdrbrg WDYT?
```suggestion
		if err != nil && !strings.Contains(err.Error(), "Unable to remove nonexistent key:") {
```